packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -83,14 +81,13 @@ const JurisdictionCard = ({
             return row.clickableDescription.map(description => {
                 if (description.type === 'link') {
                     return (
-                        <StaticLink
-                            className='text-brand-red-light '
+                        <Link
```

Comment:
We are using StaticLink component from `@deriv/library`. It has the same features as the one in `@deriv/quill-design` however it retains the legacy design so you wouldn't have to overwrite the stylings. You can try it out see if there's any difference. If so please replace but if it doesn't satisfy your criteria then this would be fine. 😇

Reply:
The styles are different from the StaticLink component.